pkgs/top-level/all-packages.nix

```diff
@@ -31406,6 +31406,10 @@ with pkgs;
     jack = libjack2;
   };
 
+  libmpg123 = mpg123.override {
```

Comment:
I think this should rather be an extra output or passthru rather than another top level attr